front-end/src/domains/home/components/home-hero-section/hero-section.tsx

```diff
@@ -0,0 +1,24 @@
+import { FC } from "react";
+import { Link } from "react-router-dom";
+import "./hero-section.scss";
+import placeholderimage from "./placeholder-assets/user-image.jpg";
+
+export const HeroSection: FC = () => (
+  <section className="hero-section">
+    <div className="hero-section-image-div"></div>
+    <img className="hero-section-image" src={placeholderimage}></img>
+    <p className="hero-section-image-focus">Image focus</p>
+    <div className="hero-section-div">
+      <p className="hero-section-title">
+        Play online in browser <br />
+        (or tagline goes here!)
+      </p>
+      <Link to="/game">
+        <button className="hero-section-button">PLAY NOW</button>
+      </Link>
+      <a className="hero-section-signin" href="">
```

Comment:
Not a blocking change, but if you’re feeling spicy you can link to the login / signup page here as well!

you don’t even need to make it an <a> tag 😉